src/opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -79,6 +79,7 @@
 #include <opm/input/eclipse/Parser/ParserKeywords/B.hpp>
 #include <opm/input/eclipse/Parser/ParserKeywords/C.hpp>
 #include <opm/input/eclipse/Parser/ParserKeywords/E.hpp>
+#include <opm/input/eclipse/Parser/ParserKeywords/F.hpp>
```

Comment:
No objection, but do we use the `F` header here?

Reply:
leftover from previous implementation. will remove.